proto/umee/incentive/v1/genesis.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+package umeenetwork.umee.incentive.v1;
+
+option go_package = "github.com/umee-network/umee/v3/x/incentive";
+
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "umee/incentive/v1/incentive.proto";
+
+// GenesisState defines the x/incentive module's genesis state.
+message GenesisState {
+  Params                    params               = 1 [(gogoproto.nullable) = false];
+  repeated IncentiveProgram completed_programs   = 2 [(gogoproto.nullable) = false];
```

Comment:
hmm, maybe we can add it to v2, once we will have proper indexer?

Reply:
Since each one must be passed by governance and the `IncentiveProgram` structure is small, I don't think storing them in state is too threatening. It's better to have a way to audit what programs have existed in the past.